waltz-common/src/main/java/com/khartec/waltz/common/StringUtilities.java

```diff
@@ -113,4 +118,19 @@ public static int length(String str) {
                 ? 0
                 : str.length();
     }
+
+
+    public static String join(Collection<?> values, String separator) {
+        return values.stream()
+                .filter(v -> v != null)
+                .map(v -> v.toString())
+                .collect(Collectors.joining(separator));
+    }
+
+
+    public static <T> List<T> split(String value, String separator, Function<String, T> fromString) {
```

Comment:
Does't allow me to use `identity` since the function goes from `String -> T`

```
public static <T> List<T> splitThenMap(String str, String separator, Function<String, T> itemTransformer) {
        if (str == null) { return Collections.emptyList(); }
        if (itemTransformer == null) {
            itemTransformer = s -> (T) s;
        }

        return Arrays.stream(str.split(separator))
                .map(itemTransformer)
                .collect(toList());
    }
```